src/everest/optimizer/everest2ropt.py

```diff
@@ -285,14 +139,19 @@ def _parse_objectives(objective_functions: list[ObjectiveFunctionConfig], ropt_c
         ropt_config["function_transforms"] = transforms
 
 
-def _parse_input_constraints(
-    input_constraints: list[InputConstraintConfig] | None,
-    ropt_config,
-    formatted_names,
-):
-    if not input_constraints:
+def _parse_input_constraints(ever_config: EverestConfig, ropt_config):
```

Comment:
I remember there was an issue some time back refactoring code to avoid using the full Everest config as function arguments.

Reply:
I don't know if it is an issue, but since that seems to be our policy now, I will fix it.